cli/cmd/project/refresh.go

```diff
@@ -0,0 +1,67 @@
+package project
+
+import (
+	"fmt"
+
+	"github.com/rilldata/rill/cli/pkg/cmdutil"
+	"github.com/rilldata/rill/cli/pkg/config"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func RefreshCmd(cfg *config.Config) *cobra.Command {
+	var project, path string
+	var refreshSources []string
+
+	refreshCmd := &cobra.Command{
+		Use:               "refresh [<project-name>]",
+		Args:              cobra.MaximumNArgs(1),
+		Short:             "Refresh project",
+		PersistentPreRunE: cmdutil.CheckChain(cmdutil.CheckAuth(cfg), cmdutil.CheckOrganization(cfg)),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+
+			client, err := cmdutil.Client(cfg)
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+
+			if len(args) > 0 {
+				project = args[0]
+			}
+
+			if !cmd.Flags().Changed("project") && len(args) == 0 && cfg.Interactive {
+				var err error
+				project, err = inferProjectName(ctx, client, cfg.Org, path)
+				if err != nil {
+					return err
+				}
+			}
+
+			resp, err := client.GetProject(ctx, &adminv1.GetProjectRequest{
+				OrganizationName: cfg.Org,
+				Name:             project,
+			})
+			if err != nil {
+				return err
+			}
+
+			_, err = client.TriggerRefreshSources(ctx, &adminv1.TriggerRefreshSourcesRequest{DeploymentId: resp.ProdDeployment.Id, Sources: refreshSources})
+			if err != nil {
+				return err
+			}
+
+			fmt.Printf("Triggered refresh. To see status, run `rill project status --project %s`.\n", project)
+
+			return nil
+		},
+	}
+
+	refreshCmd.Flags().SortFlags = false
+	refreshCmd.Flags().StringVar(&project, "project", "", "Project name")
+	refreshCmd.Flags().StringVar(&path, "path", ".", "Project directory")
+	refreshCmd.Flags().StringSliceVar(&refreshSources, "refresh-source", nil, "Refresh specific source(s)")
```

Comment:
Can we make the flag here just `--source` – I think that would enable `rill project refresh --source src_a --source src_b` to refresh two sources